### DISCUSSION TITLE
Isolation of Ionic Capacitor Apps

### DISCUSSION THREAD

**Discussion by TomStroemer** [ionic-team/capacitor#5880] [Q&A] [answered]
We are building an Ionic App with capacitor. As far as I understand the architecture this starts a webserver in the app to serve the web resources (Angular) so they can be loaded in the webview.

My question is: How does isolation work, when running multiple Ionic Apps? Is the webserver only accessible for the app? Why are there no port conflicts?

Is there any detailed documentation for such topics?

Thanks in advance!

**Comment by jcesarmobile** [comment, marked as answer]
It’s not a real web server, so there is no conflict with other apps. In the past iOS had a real web server and it had conflicts, so we removed it.
It uses WKURLScheme handler instead that intercepts requests to custom schemes
https://developer.apple.com/documentation/webkit/wkurlschemehandler
